CMakeLists.txt

```diff
@@ -183,8 +183,14 @@ add_subdirectory(doc)
 
 install(DIRECTORY share/castxml/ DESTINATION "${CastXML_INSTALL_DATA_DIR}")
 
+if(NOT DEFINED CastXML_INSTALL_LICENSE OR CastXML_INSTALL_LICENSE)
+  set(_license_install LICENSE)
+else()
+  set(_license_install)
+endif()
+
 install(FILES
-  "LICENSE"
+  ${_license_install}
```

Comment:
Interesting.  I haven't seen such a switch in other projects.  What is special/new here?

Reply:
On second thought, this switch is overkill. Downstream can just use the patch. I will remove it.